src/client/renderer/r_types.h

```diff
@@ -858,6 +895,7 @@ typedef struct {
 	uint32_t num_mesh_tris;
 
 	_Bool update; // inform the client of state changes
+	_Bool scissor_enabled;
```

Comment:
There is no scissor enable bit in r_state_t, but you're right in that it should be there.